cms/templates/settings.html

```diff
@@ -361,7 +361,7 @@ <h2 class="title-2">${_("Introducing Your Course")}</h2>
               </ol>
             </section>
 
-          % if about_page_editable:
+          % if about_page_editable or is_prerequisite_courses_enabled or settings.FEATURES.get('ENTRANCE_EXAMS'):
```

Comment:
I think we should create (or re-use) an operation for checking the Entrance Exams feature -- ie, "is_entrance_exams_enabled()", rather than adding new settings references in the template layer.